cmd-export.c

```diff
@@ -122,7 +127,7 @@ int cmd_export(int argc, char **argv)
 		}
 	}
 
-	printf("url,username,password,extra,name,grouping,fav\r\n");
+	printf("id,name,group,groupname,fullname,url,username,password,note,last_touch,last_modified_gmt,fav,attachpresent\r\n");
 
```

Comment:
Ok, I wasn't sure if there was a spec that we had to adhere to.  I'd suppose that import should be able to round-trip export (though this can't be used to entirely re-create an account in lastpass - there are still attachments to be considered).    

What's 'website import'?  If that's a spec we need to adhere to, perhaps export could take a `--full` or `--extended` flag to get this output?

I would be happy to work on updating import to accept (only) the same format that export produces.

To answer the second question, yes, if there was a more machine readable data format that included all of the account information (esp notes) then my use case would be fully supported.